test/manual/amp-carousel.amp.html

```diff
@@ -89,5 +89,11 @@
     <amp-img src="https://lh3.googleusercontent.com/5rcQ32ml8E5ONp9f9-Rf78IofLb9QjS5_0mqsY1zEFc=w400-h300-no" width=400 height=300></amp-img>
     <amp-img src="https://lh3.googleusercontent.com/Z4gtm5Bkxyv21Z2PtbTf95Clb9AE4VTR6olbBKYrenM=w400-h300-no" width=400 height=300></amp-img>
   </amp-carousel>
+
+  <amp-carousel id="carousel-4" width=400 height=300 type=slides controls indicators>
```

Comment:
I am interested in the carousel, I implemented the dot system to get familiar with the AMP extensions. Knowing that the dot system isn't functional in Mobile, I happened to implement a mini previews system in the bottom of the carousel